_data/sites.yml

```diff
@@ -33,6 +33,7 @@
   size: 142
   last_checked: 2022-02-03
 
+
```

Comment:
Just for the sake of keeping a consistent formatting: could you remove this dangling whitespace?